### PR TITLE
Space visual issues fix

### PR DESCRIPTION
**Pull request by alexxpetrov**
Closes:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1799

* Remove 'assets/images/sparkle-bg-2.png' as the default background ( didn't find this asset )
* The background for the element should be light grey #e6e6e6 or its dark/light mode equivalent (not white) ( used theme color )
* fixed chat container moves space background
* fixed chat side bar absolute/relative location (stuck to bottom of the body when footer is seen/follows scroll when footer isn't seen )

### CODE REVIEW

File: src/components/attendee/AttendeeLayout/AttendeeLayout.module.scss [hunk 1]
```diff
@@ -53,9 +54,10 @@ $-z-index--popover: layer.level($-z-index--ui);
   }
 }
 
-.LayerUiAbsolute {
-  position: absolute;
-  top: 0px;
+.LayerUiRelative {
```
Comment (goran-peoski-work) [inline src/components/attendee/AttendeeLayout/AttendeeLayout.module.scss]: Wait, are we renaming this now? How about `layer`? Do we need it complicated?

Comment (goran-peoski-work) [inline src/components/attendee/AttendeeLayout/AttendeeLayout.module.scss]: Wait, are we renaming this now? How about `layer`? Do we need it complicated?

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/components/attendee/AttendeeLayout/AttendeeLayout.module.scss (hunk 1 above)
Comment (alexxpetrov) [reply src/components/attendee/AttendeeLayout/AttendeeLayout.module.scss]: renamed into `relative`

**Pull request merged**